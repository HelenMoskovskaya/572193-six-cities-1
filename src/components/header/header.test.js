import React from 'react';
import renderer from 'react-test-renderer';
import Header from './header.jsx';

const mockUserData = {
  id: 1,
  email: `Oliver.conner@gmail.com`,
  name: `Oliver.conner`,
  avatar_url: `img/1.png`,
  is_pro: false
};

it(`App correcrly renders`, () => {
  const tree = renderer

  .create(<Header
    isAuthorizationRequired={true}
    userData={mockUserData}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});
