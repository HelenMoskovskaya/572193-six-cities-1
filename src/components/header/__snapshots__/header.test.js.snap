// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App correcrly renders 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__wrapper"
    >
      <div
        className="header__left"
      >
        <a
          className="header__logo-link"
          href="main.html"
        >
          <img
            alt="6 cities logo"
            className="header__logo"
            height="41"
            src="img/logo.svg"
            width="81"
          />
        </a>
      </div>
      <nav
        className="header__nav"
      >
        <ul
          className="header__nav-list"
        >
          <li
            className="header__nav-item user"
          >
            <a
              className="header__nav-link header__nav-link--profile"
              href="#"
            >
              <div
                className="header__avatar-wrapper user__avatar-wrapper"
              >
                <img
                  className="user__avatar"
                  src="https://es31-server.appspot.com/six-citiesundefined"
                />
              </div>
              <span
                className="header__login"
              >
                Oliver.conner@gmail.com
              </span>
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </div>
</header>
`;
